check/doctest

```diff
@@ -0,0 +1,18 @@
+#!/usr/bin/env bash
+
+################################################################################
+# Runs python doctest on python source files in the repository.
+#
+# See also:
+#     https://docs.python.org/3/library/doctest.html
+#
+# Usage:
+#     check/doctest
+################################################################################
+
+# Get the working directory to the repo root.
+cd "$( cd "$( dirname "${BASH_SOURCE[0]}" )" && pwd )"
+cd "$(git rev-parse --show-toplevel)"
+
+python_source_files=(`find cirq | grep "\.py$"`)
+python -m doctest "${python_source_files[@]}"
```

Comment:
Putting all the files on the command line risks exceeding the maximum command length.  Also user arguments aren't passed to doctest.  Just use find: `find cirq -type f -iname '*.py' | xargs python -m doctest $@`.

Reply:
Not having user arguments was on purpose (note it's not mentioned in the usage instructions). Used your line without the $@.